examples/zookeeper_example.rs

```diff
@@ -91,39 +91,40 @@ fn zk_example() {
 
     let zk_arc = Arc::new(zk);
 
-    let mut pcc = PathChildrenCache::new(zk_arc.clone(), "/").unwrap();
-    match pcc.start() {
-        Err(err) => {
-            println!("error starting cache: {:?}", err);
-            return;
-        }
-        _ => {
-            println!("cache started");
-        }
-    }
-
-    let (ev_tx, ev_rx) = mpsc::channel();
-    pcc.add_listener(move |e| ev_tx.send(e).unwrap());
-    thread::spawn(move || {
-        for ev in ev_rx {
-            println!("received event {:?}", ev);
-        }
-    });
+    // let mut pcc = PathChildrenCache::new(zk_arc.clone(), "/").unwrap();
+    // match pcc.start() {
+    //     Err(err) => {
+    //         println!("error starting cache: {:?}", err);
+    //         return;
+    //     }
+    //     _ => {
+    //         println!("cache started");
+    //     }
+    // }
+
+    // let (ev_tx, ev_rx) = mpsc::channel();
+    // pcc.add_listener(move |e| ev_tx.send(e).unwrap());
+    // thread::spawn(move || {
+    //     for ev in ev_rx {
+    //         println!("received event {:?}", ev);
+    //     }
+    // });
```

Comment:
somehow my comment got lost: should we just delete this for now? (remains in git)